R/render.r

```diff
@@ -482,7 +499,18 @@ check_bootswatch_theme <- function(bootswatch_theme, bs_version, pkg) {
 }
 
 pkgdown_bslib_defaults <- function() {
-  list(`navbar-nav-link-padding-x` = "1rem")
+  list(
+    `navbar-nav-link-padding-x` = "1rem",
+    `primary` = "#0054AD",
+    `secondary` = "#e9ecef",
+    `border-color` = "#e5e5e5",
+    `navbar-light-color` = "rgba(0,0,0,0.7)",
+    `navbar-light-hover-color` = "rgba(0,0,0,0.9)",
+    `border-width` = "1px",
+    `code-bg` = "#f8f8f8",
+    `code-color` = "#333",
+    `fu-color` = "#4758AB"
```

Comment:
this "just works".

Reply:
"fu" 🤣

Reply:
Besides this being funny :grin: I am not sure yet how to handle these colors cf https://github.com/r-lib/pkgdown/issues/1696#issuecomment-845272050